src/linux-mingw-dependencies/linux-mingw-dependencies.csproj

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I think this file should go into `build-tools`, not `src`. The *outputs* of this project will not be redistributed; instead, they'll be used in order to build e.g. libmono, which is redistributed.